index.bs

```diff
@@ -1075,12 +1112,68 @@ To <dfn export for="SendStream" lt="create|creating">create</dfn> a
 1. Let |stream| be a [=new=] {{SendStream}}, with:
     : [=[[InternalStream]]=]
     :: |internalStream|
-1. [=WritableStream/Set up=] |stream| with TBD.
-1. [=set/Add=] |stream| to |transport|'s [=[[SendStreams]]=].
+    : [=[[Transport]]=]
+    :: |transport|
+1. Let |writeAlgorithm| be an action that [=writes=] |chunk| with |stream|, given |chunk|.
+1. Let |closeAlgorithm| be an action that [=closes=] |stream|.
+1. Let |abortAlgorithm| be an action that [=aborts=] |stream|.
```

Comment:
Maybe you need to specify `reason` here?

Reply:
Fixed.